WordPressVIPMinimum/Sniffs/SVG/HTMLCodeSniff.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * WordPressVIPMinimum Coding Standard.
+ *
+ * @package VIPCS\WordPressVIPMinimum
+ */
+
+namespace WordPressVIPMinimum\Sniffs\SVG;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use PHP_CodeSniffer\Util\Tokens;
+
+/**
+ * This function generates an error when
+ * <?php or <? is found in a SVG file.
+ *
+ * The aim is to bring this to the notice of
+ * reviewers, so they can make sure the PHP-code
+ * is safe. The reason for this is that SVG files
+ * are often included in PHP files, but manually
+ * scanning SVG files takes alot of time and is prone
+ * to errors.
+ */
+class HTMLCodeSniff implements Sniff {
+
+	/**
+	 * Returns the token types that this sniff is interested in.
+	 *
+	 * We want everything function-related.
+	 *
+	 * @return array(int)
+	 */
+	public function register() {
+		return array(
+			T_OPEN_TAG,
```

Comment:
There is also `T_OPEN_TAG_WITH_ECHO` checking for `<?=` which should be part of this sniff, imho.